.config/nvim/lua/plugins/codecompanion.lua

```diff
@@ -363,6 +363,15 @@ The current projects structure looks like #{ls} and the open file is #{buffer}.
           },
           spinner = {},
           reasoning = { callback = "codecompanion._extensions.reasoning", opts = { enabled = true } },
+          rules_loader = {
```

Comment:
Please add the variables ls and rules to the agent mode prompt. Make a nice sentence when adding them so that it all makes sense